docs/tools/paywall.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Paywall
+description: >-
+  Guide to the Paywall JavaScript library.
+---
+# Paywall
+
+The Paywall is a lightweight JavaScript library which can track state and emits
```

Comment:
It's not that lightweight but we need to make it lighter :D

Reply:
Changed it to simple, which is true now :-)